app/services/katello/pulp/repository.rb

```diff
@@ -149,29 +149,32 @@ def proxy_host_importer_value
       end
 
       def refresh
-        update_or_associate_importer
+        tasks = update_or_associate_importer
         update_or_associate_distributors
         remove_unnecessary_distributors
+        tasks
```

Comment:
Do we not want to track distributor tasks if we are doing so for importer tasks?

Reply:
I based this on older code which didn't seem to indicate that distributor associate/delete/update tasks were async, but i guess i should double check that :)

Reply:
turns out some (but not all) of those distributor tasks do generate tasks, added additional tracking